SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <title>SSO</title>
-</head>
-<body>
-    <div data-role="page" class="page type-interior pluginConfigurationPage esqConfigurationPage">
-        <div data-role="content">
-            <div class="content-primary">
-                <a href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md">Review the documentation. This plugin is configured via the API.</a>
+  </head>
+  <body>
+    <div
+      id="sso-config-page"
+      data-role="page"
+      class="page type-interior pluginConfigurationPage esqConfigurationPage"
+      data-controller="__plugin/SSO-Auth.js"
+    >
+      <div data-role="content">
+        <div class="content-primary">
+          <a
```

Comment:
Can the contrast of this be improved as well as the padding be fixed?

Reply:
The string probably should also be changed (no longer just configured via API)

Reply:
Should become a help or advanced config button

Reply:
> Can the contrast of this be improved as well as the padding be fixed?

If you mean the link to readme, yes, the contrast is bad - I think I'll change it to a help button, and add a disclaimer that this is in alpha, doesn't have SAML, and to check the help for more info. 

As for padding, I'll add a proper title, and between that and the disclaimer it should look more polished

LDAP for reference:
```html
                    <div class="verticalSection verticalSection-extrabottompadding">
                        <div class="sectionTitleContainer flex align-items-center">
                            <h2 class="sectionTitle">LDAP Settings:</h2>
                            <a is="emby-button" class="raised button-alt headerHelpButton" target="_blank" href="https://github.com/jellyfin/jellyfin-plugin-ldapauth">${Help}</a>
                        </div>
                        <p><i>Note:</i> Making changes to this configuration requires a restart of Jellyfin.</p>
```

![image](https://user-images.githubusercontent.com/39424834/165016898-f95ac144-a07b-48be-aebb-9b6bf75d5cdf.png)

Reply:
Resolved by https://github.com/9p4/jellyfin-plugin-sso/pull/18/commits/5112bb13afe9943aad8784e551d7277870ddeaae